elastalert/ruletypes.py

```diff
@@ -376,6 +391,10 @@ def __init__(self, *args):
         self.first_event = {}
         self.skip_checks = {}
 
+        self.field_value = None
```

Comment:
self.field_value = self.rules.get('field_value')

This is the exact same as the three current lines